### PR TITLE
Cleaning up extra printed messages added during debugging

### PR DESCRIPTION
**Pull request by gregmedd**
While debugging the RPC timeout issue, several extra messages were added and the reply data format were changed. Changing these back so the example is cleaner and outputs the expected time value.

### CODE REVIEW

File: rpc/src/main_rpc_server.cpp [hunk 1]
```diff
@@ -51,27 +51,30 @@ class RpcListener : public UListener {
...
+
+            UPayload responsePayload(
+                    reinterpret_cast<const uint8_t*>(&currentTimeMilli),
+                    sizeof(currentTimeMilli),
+                    UPayloadType::VALUE);
+
             /* Build response attributes - the same UUID should be used to send the response 
              * it is also possible to send the response outside of the callback context */
+            auto builder = UAttributesBuilder::response(
+                    message.attributes().source(),
```
Comment (stevenhartley) [inline rpc/src/main_rpc_server.cpp:68]: source is sink and sink is source so flip these two

File: rpc/src/main_rpc_server.cpp [hunk 2]
```diff
@@ -51,27 +51,30 @@ class RpcListener : public UListener {
...
+                    message.attributes().sink(),
+                    UPriority::UPRIORITY_CS0,
```
Comment (stevenhartley) [inline rpc/src/main_rpc_server.cpp:70]: This needs to be message.attributes.priority().

**Review by stevenhartley — CHANGES_REQUESTED** [review]

File: rpc/src/main_rpc_server.cpp (hunk 1 above)
Comment (gregmedd) [reply rpc/src/main_rpc_server.cpp]: fixed

File: rpc/src/main_rpc_server.cpp (hunk 2 above)
Comment (gregmedd) [reply rpc/src/main_rpc_server.cpp]: fixed

**Review by stevenhartley — APPROVED** [review]

File: rpc/src/main_rpc_server.cpp [hunk 3]
```diff
@@ -51,27 +51,30 @@ class RpcListener : public UListener {
...
+                    reinterpret_cast<const uint8_t*>(&currentTimeMilli),
+                    sizeof(currentTimeMilli),
+                    UPayloadType::VALUE);
+
             /* Build response attributes - the same UUID should be used to send the response 
              * it is also possible to send the response outside of the callback context */
+            auto builder = UAttributesBuilder::response(
+                    message.attributes().sink(),
+                    message.attributes().source(),
+                    message.attributes.priority(),
```
Comment (stevenhartley) [inline rpc/src/main_rpc_server.cpp:70]:
```suggestion
                    message.attributes().priority(),
```

**Review by stevenhartley — CHANGES_REQUESTED** [review]
There is a bug for priority

File: rpc/src/main_rpc_server.cpp (hunk 3 above)
Comment (gregmedd) [reply rpc/src/main_rpc_server.cpp]: Fixed

**Review by stevenhartley — APPROVED** [review]

**Pull request merged**